scripts/check_license.sh

```diff
@@ -1,10 +1,29 @@
 #!/bin/sh
+#
+# Copyright 2018 The Prometheus Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+check_license() {
+  local file=""
+  for file in $(find . -type f -iname '*.go' ! -path './vendor/*'); do
+    head -n3 "${file}" | grep -Eq "(Copyright|generated|GENERATED)" || echo "  ${file}"
+  done
+}
+
+licRes=$(check_license)
 
-licRes=$(
-for file in $(find . -type f -iname '*.go' ! -path './vendor/*'); do
-	head -n3 "${file}" | grep -Eq "(Copyright|generated|GENERATED)" || echo -e "  ${file}"
-done;)
 if [ -n "${licRes}" ]; then
-	echo -e "license header checking failed:\n${licRes}"
-	exit 255
+  echo "license header checking failed:"
+  echo "${licRes}"
+  exit 255
```

Comment:
An exit code of 1 would make more sense I think

Reply:
Yea, I have no idea why it's 255.